main/webapp/modules/core/scripts/views/data-table/data-table-view.js

```diff
@@ -36,13 +36,21 @@ function DataTableView(div) {
 
   this._gridPagesSizes = JSON.parse(Refine.getPreference("ui.gridPaginationSize", null));
   this._gridPagesSizes = this._checkPaginationSize(this._gridPagesSizes, [ 5, 10, 25, 50 ]);
```

Comment:
These two lines are obsolete as well, correct?

Reply:
Actually, on further review, there's a whole bunch of pagination related code that appears to be obsolete: lines 180-244, `onClick*Page` functions, etc.

Reply:
Yup, I will get to removing them.